src/main/kotlin/com/jfrog/conan/clionplugin/services/ConanPluginInit.kt

```diff
@@ -3,10 +3,12 @@ package com.jfrog.conan.clionplugin.services
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.startup.ProjectActivity
 import com.jfrog.conan.clionplugin.conan.ConanPluginUtils
+import com.intellij.openapi.options.advanced.AdvancedSettings
 
 
 class ConanPluginInit : ProjectActivity {
     override suspend fun execute(project: Project) {
         ConanPluginUtils.downloadCMakeProvider()
+        AdvancedSettings.setBoolean("cmake.reload.profiles.sequentially", true)
```

Comment:
I think maybe we should do this when the user adds support for Conan in the project, by hitting the + button or when having Conan support enabled for more than one config? And we probably should warn about this being changed and how to revert that.